src/pages/[platform]/prev/build-a-backend/storage/import/index.mdx

```diff
@@ -67,8 +67,12 @@ By default, Amplify Libraries assumes that S3 buckets are configured with the fo
 - `protected/{user_identity_id}/` - Readable by all users, but writable only by the creating user
 - `private/{user_identity_id}/` - Only accessible for the individual user
 
+<InlineFilter filters={[]}>
```

Comment:
Q: what's the purpose of an empty filter?

Reply:
It was just flutter there before so I removed flutter from it. I'm just going to delete the line since that'll be cleaner.